utils.js

```diff
@@ -39,7 +39,20 @@ function get_min_tag_sizes({ fee_rate }) {
       }, {});
 }
 
+function get_included_tags() {
```

Comment:
Ahh we should also match excluded_fees by allowing the _HIGH_FEE and _MEDIUM_FEE thresholds